raml2markdown/slate/index.md

```diff
@@ -48,30 +48,26 @@ Our [Developer Portal](https://developers.oftrust.net) is your one-stop-shop. Fr
 
 ### End-to-end developer experience
 
-APIs play crucial role in our end-end-end developer experience from integrating data to creating valuable applications. API -first experience and consistent APIs are important to us and thus we have created (work in progress) [API Design Guide](https://platformoftrust.gitbook.io/api-design-guide/) to offer guidance for our distributes API development teams. 
+APIs play crucial role in our end-end-end developer experience from integrating data to creating valuable applications. API -first experience and consistent APIs are important to us and thus we have created (work in progress) [API Design Guide](https://design.oftrust.net/) to offer guidance for our distributes API development teams. 
 
 ![End-to-end developer experience in Platform of Trust](images/dx.png)
 
-### Market place
-
-Market place is the bazaar to find more data products to use in application development. Visa versa, it is also the service where your data products are added during the integration process. 
-You can list data products in the market place with [Product API](#product-api). 
 
 # Getting started
 
 > Some instructions and tips to make your life easier (and less support requests to us): 
 
-> - **Create an account in sandbox** environment from https://world-sandbox.oftrust.net/
+> - **[Create an account in sandbox](https://login-sandbox.oftrust.net/?grant_type=authorization&response_type=code&redirect_uri=https://world-sandbox.oftrust.net/api/exchangeToken&client_id=f773dafe-20c0-4a25-aa3e-9da0b81b9304&cancel_url=https://world-sandbox.oftrust.net&reset_password_success_url=https://world-sandbox.oftrust.net/api/login&state=eyJkIjogeyJyIjogImh0dHBzOi8vd29ybGQtc2FuZGJveC5vZnRydXN0Lm5ldC9hcGkvZXhjaGFuZ2VUb2tlbiIsICJ0cyI6IDE1ODI2MTAzOTV9LCAiaCI6ICJmNmVjYTgwOTQyOGFiNzZkZDFkN2ZhNmI1MzhiZjM3OGZmOGRkZDA1In0=#sign-up)** environment
 
 > - Endpoints related code examples are constructed against **SANDBOX environment `https://api-sandbox.oftrust.net/`**. 
 
-> - In **PRODUCTION** use, change domain in api endpoints to `https://api.oftrust.net/`
+> - In **PRODUCTION** use, change domain in API endpoints to `https://api.oftrust.net/`
 
 > - To test APIs you need **to get needed Bearer Token** See [Authentication section](#use-bearer-token-and-how-to-get-it)
 
-> If you found a bug or missing information in the documentation, contact us at dev@oftrust.net or create an [issue in Github](https://github.com/PlatformOfTrust/docs/issues/new). 
+> If you found a bug or missing information in the documentation, create an [issue in Github](https://github.com/PlatformOfTrust/collected-feedback/issues/new?assignees=&labels=APIs&template=api-wishlists.md&title=Wish+for+the+API). 
```

Comment:
"wish for the API" means someone wants to request a completely new API, right? Maybe "wish changes for the API" or "suggestions for the API", or something like that.

Reply:
Then someone has to do a new template for this. I changed the title of the issue to "Feedback about getting started". If this is satisfactory, then this is solved.